CONTRIBUTING.txt

```diff
@@ -170,6 +170,49 @@ Once you've fixed all merge conflicts, do::
    <https://scikit-image.org/docs/dev/gitwash/development_workflow.html#rebasing-on-trunk>`__,
    but we squash and merge most PRs either way.
 
+Build environment setup
+-----------------------
+
+Once you've cloned your fork of the scikit-image repository,
+you should setup a Python development environment tailored for scikit-image.
+You may choose the environment manager of your choice.
+Here we provide instructions for two popular environment managers:
+``venv`` (pip based) and ``conda`` (Anaconda or Miniconda).
+
+venv
+====
+When using ``venv``, you may find the following bash commands useful::
+
+  # Create a virtualenv named ``skimage-dev`` that lives in the directory of
+  # the same name
+  python -m venv skimage-dev
+  # Activate it
+  source skimage-dev/bin/activate
+  # Install all development and runtime dependencies of scikit-image
+  pip install -r <(cat requirements/*.txt)
+  # Build and install scikit-image from source
+  pip install -e .
+  # Test your installation
+  pytest skimage
+
+conda
+=====
+
+When using conda, you may find the following bash commands useful::
+
+  # Create a conda environment named ``skimage-dev``
+  conda create --name skimage-dev
+  # Activate it
+  conda activate skimage-dev
+  # Install all development and runtime dependencies of scikit-image
+  conda install `for i in requirements/{default,build}.txt; do echo -n " --file $i "; done`
```

Comment:
It will, probably, be better to harmonize the dependency installation instructions for `pip` and `conda` sections (i.e. consider either all or (default,build) req-files).

Reply:
everything installs on conda-forge, but not on defaults. Many people aren't on conda forge, so I feel like this is a just middle to hit, without asking people to go bleeding edge (on conda-forge) and messing the rest of their conda config.

Reply:
Maybe give a hint then that these dependency lists are not actually complete? E.g., change:
```
# Install all development and runtime dependencies of scikit-image
```
to
```
# Install major development and runtime dependencies of scikit-image
# (the rest can be installed from conda-forge or pip, if needed)
```

Reply:
Thanks for the suggestion!